### PR TITLE
Upgrade pegasus dependency

### PR DESCRIPTION
**Pull request by zxcware**

### CODE REVIEW

File: gobblin-rest-service/gobblin-rest-server/build.gradle [hunk 1]
```diff
@@ -59,6 +59,7 @@ buildscript {
 
     dependencies {
         classpath group: 'com.linkedin.pegasus', name: 'gradle-plugins', version: pegasusVersion
+        classpath externalDependency.jdkTools
```
Comment (ibuenros) [inline gobblin-rest-service/gobblin-rest-server/build.gradle]: Why is this needed?

Comment (zxcware) [inline gobblin-rest-service/gobblin-rest-server/build.gradle]: pegasus version 11.0.0 has a bug. It fails to add tools.jar into the classpath. They're working on a clean fix. This is a temporary fix for all the open sources that rely on restli version 7+. I moved the fix to the root `build.script`. Please see the latest diff.
